splice_machine/src/main/java/com/splicemachine/derby/stream/output/insert/InsertTableWriterBuilder.java

```diff
@@ -243,4 +247,16 @@ public static InsertTableWriterBuilder getInsertTableWriterBuilderFromBase64Stri
     public String getInsertTableWriterBuilderBase64String() throws IOException, StandardException {
         return Base64.encodeBase64String(SerializationUtils.serialize(this));
     }
+
+
+    @Override
+    public DataSetWriterBuilder loadReplaceMode(boolean load_replace_mode) {
+        this.loadReplaceMode = load_replace_mode;
```

Comment:
parameter load_replace_mode  name not in Java style

Reply:
done